rules/library.bzl

```diff
@@ -423,7 +423,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
                 objc_private_hdrs.append(f)
             else:
                 objc_hdrs.append(f)
-        elif f.endswith((".m", ".mm", ".c")):
+        elif f.endswith((".m", ".mm", ".c", ".s", ".S")):
```

Comment:
Do we need/want those in `non_arc_srcs` as well? Legit question, using this PR to learn more about this use case (.S files) 😄

Reply:
no, I think `non_arc_srcs` is only relevant on `.m` or `.mm` files